dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -992,6 +1016,13 @@ BlockInputStreamPtr Segment::getInputStream(
             /*read_pack*/ {});
         pack_filter_results.push_back(result);
     }
+    const auto & res_group_name = dm_context.scan_context->resource_group_name;
+    if (likely(read_mode == ReadMode::Bitmap && !res_group_name.empty()))
+    {
+        const auto keyspace_id = dm_context.scan_context->keyspace_id;
+        auto bytes = estimatedBytesOfInternalColumns(dm_context, segment_snap, pack_filter_results, start_ts);
+        LocalAdmissionController::global_instance->consumeBytesResource(keyspace_id, res_group_name, bytesToRU(bytes));
```

Comment:
Add some comments about why we need to consumeBytesResource by estimate in advance

Reply:
Add in https://github.com/pingcap/tiflash/pull/10378/commits/55486732dcd931aabd0d8535b83bf2e2e8b019aa